app/lib/card_screenshotter/members.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+include Rails.application.routes.url_helpers
+
+module CardScreenshotter
+  class Members
+    def self.update_screenshots
+      driver = CardScreenshotter::Utils.open_headless_driver
+
+      card_width = 600
+      card_height = 350
+      save_path = Rails.root.join("public/cards/member_policy_vote")
+
+      PolicyPersonDistance.find_each do |ppd|
+        person = ppd.person
+        policy = ppd.policy
+        url = "http://#{ActionMailer::Base.default_url_options[:host]}#{person_policy_path_simple(person, policy)}?card=true"
+        file_name = person_policy_path_simple(person, policy).gsub("/", "_")
```

Comment:
What's an example url for one of the generated images and the corresponding url for the person policy page that it's related to?

Reply:
URL: `https://theyvoteforyou.org.au/people/senate/victoria/raff_ciccone/policies/86`
file name: `_people_senate_victoria_raff_ciccone_policies_86.png`

Reply:
I'm specifically asking for the **url** that you would access the generated image from.

Reply:
I don't know, I have not been able to serve images that are in `public` directory to see how the URL would look like. How would I do that, I cannot find a resource to link images outside of the asset pipeline

Reply:
Rails should just serve stuff from `public` in development. In production `public` will get served by nginx. So, for example if you go to http://localhost:3000/robots.txt it will serve the contents of `public/robots.txt`. Try it.

Reply:
Oh, the URL would look like this `http://localhost:3000/cards/member_policy_vote/_people_senate_act_kate_lundy_policies_1.png`

Reply:
Thanks. In that case what would it take for the URL for the generated image to be?
https://theyvoteforyou.org.au/cards/people/senate/victoria/raff_ciccone/policies/86.png

Reply:
Since you cannot have the `/` character in a file name you would have to create the directory `public/cards/people/senate/victoria/raff_ciccone/policies/` so that the URL can have the `/cards/people/senate/victoria/raff_ciccone/policies/` and the image name would `86.png`

Reply:
I have made that change, i'll push it now.